common/marshal/marshal.go

```diff
@@ -187,8 +187,12 @@ func MarshalJsonBlock(b *types.SerializedBlock, inclTx bool, fullTx bool,
 		{Key:"stateRoot", Val:head.StateRoot.String()},
 		{Key:"bits", Val:strconv.FormatUint(uint64(head.Difficulty), 16)},
 		{Key:"difficulty", Val:head.Difficulty},
-		{Key:"nonce", Val:head.Nonce},
-		{Key:"timestamp", Val:head.Timestamp.Format(time.RFC3339)},
+		{Key:"pow", Val : map[string]interface{}{
+			"nonce":head.Pow.GetNonce(),
+			"pow_type":pow.PowMapString[head.Pow.GetPowType()],
```

Comment:
Please unify the JSON namings, use `camelCase` (see https://golang.org/doc/effective_go.html#mixed-caps _convention in Go is to use MixedCaps or mixedCaps rather than underscores to write multiword names_.)

Reply:
fixed